### ISSUE TITLE
Broken links to deno.land/x/windmill

### ISSUE THREAD

**Issue by ralphwest1** [windmill-labs/windmilldocs#607]
in file: `docs/core_concepts/11_persistent_storage/within_windmill.mdx` links to `https://deno.land/x/windmill/mod.ts?s=setResource` and `https://deno.land/x/windmill/mod.ts?s=getResource` are broken and don't work.

- https://deno.land/x/windmill@v1.326.0/mod.ts?s=setResource
![image](https://github.com/windmill-labs/windmilldocs/assets/23242756/acd17c0d-e2e8-467e-b1a9-e4c2ade888ea)

These are on line 158 and 159 of the file

```txt
157  - `setResource(value: any, path?: string, initializeToTypeIfNotExist?: string)`: which sets a resource at a given path. This is
158    equivalent to `setState` but allows you to set an arbitrary path and chose a type other than state if wanted. [See API](https://deno.land/x/windmill/mod.ts?s=setResource).
159  - `getResource(path: string)`: gets a resource at a given path. [See API](https://deno.land/x/windmill/mod.ts?s=getResource).
```

Not sure if this a temporary `deno.land` thing or if they should be updated to point at `jsr.io`. 

I'm happy to file a pull request to update these links and any other broken `deno.land` links if you advise on where they should be pointing.

**Comment by rubenfiszel** [comment]
Correct, we will indeed make jsr the new official documentation of the typescript client. We would welcome any PRs! Thanks a lot

**Comment by ralphwest1** [comment]
No problem, When I have a little time, I'll submit a pull request to update all `deno.land` links on all pages.
